db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -917,15 +917,33 @@ Status DBImpl::CompactRange(const CompactRangeOptions& options,
                               end_with_ts);
 }
 
-Status DBImpl::IncreaseFullHistoryTsLow(ColumnFamilyData* cfd,
+Status DBImpl::IncreaseFullHistoryTsLow(ColumnFamilyHandle* column_family,
                                         std::string ts_low) {
+  ColumnFamilyData* cfd;
+  if (column_family == nullptr) {
+    cfd = default_cf_handle_->cfd();
+  } else {
+    auto cfh = static_cast_with_check<ColumnFamilyHandleImpl>(column_family);
+    assert(cfh != nullptr);
+    cfd = cfh->cfd();
+  }
+  if (cfd->user_comparator()->timestamp_size() == 0) {
+    return Status::NotSupported(
+        "Timestamp is not enabled in this column family");
+  }
+  return IncreaseFullHistoryTsLowImpl(cfd, ts_low);
```

Comment:
We should also have a check for `ts_low.empty()` and return invalid argument early.

Reply:
added